### DISCUSSION TITLE
Defined labels for "python.analysis.logLevel" setting use inconsistent capitalization

### DISCUSSION THREAD

**Discussion by erictraut** [microsoft/pylance-release#2692] [Enhancement]
Pylance's package.json defines the "python.analysis.logLevel" as an enum that allows four values: "Error", "Warning", "Info" and "Trace".

Pyright's package.json defines the same enum but uses "error", "warning", "info" and "trace". 

The code supports both, but the JSON settings editor is case-sensitive and reports a warning if you use the wrong case. We should switch one or the other for consistency. I would argue that lowercase is better because it's consistent with enums used in other settings such as "python.analysis.diagnosticSeverityOverrides" (which uses values named "none", "warning", "information" and "error").

**Comment by jakebailey** [comment]
This setting is defined in the core extension, and is used by MPLS where I believe that it is actually case sensitive. I agree that it's inconsistent and would prefer lowercase, but there's also not a super clean way of mending this that won't also annoy everyone who's already set their levels. Perhaps that doesn't matter too much; it's just a logging setting.

**Comment by judej** [comment]
Issue addressed in #353

**Comment by jakebailey** [comment]
#353 was for inconsistent values, this is about capitalization which I think we still want to do eventually anyway.

**Comment by jakebailey** [comment]
With the deprecation of MPLS next month, this setting has been moved into Pylance's extension, so this should now be unblocked if we wanted to change this.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
